test/integration/shells_int_test.go

```diff
@@ -35,59 +36,61 @@ func (suite *ShellsIntegrationTestSuite) TestShells() {
 	cp.ExpectExitCode(0)
 
 	for _, shell := range shells {
-		// Run the checkout in a particular shell.
-		cp := ts.SpawnInShell(
-			shell,
-			e2e.WithArgs("checkout", "ActiveState-CLI/small-python", string(shell)),
-		)
-		cp.Expect("Checked out project")
-		if shell != e2e.Cmd {
-			cp.ExpectExitCode(0)
-		}
+		suite.T().Run(fmt.Sprintf("using_"+string(shell)), func(t *testing.T) {
```

Comment:
Making use of printf verbs, `fmt.Sprintf("using_%s", shell)` should work. The "%s" verb will cause the fmt package to check for a String() method, or string compatibility.